cookbooks/travis_build_environment/attributes/default.rb

```diff
@@ -118,15 +118,11 @@
 default['travis_build_environment']['required_otp_release_for']['1.7.4'] = '21.1'
 default['travis_build_environment']['default_elixir_version'] = '1.7.4'
 default['travis_build_environment']['mysql']['socket'] = '/var/run/mysqld/mysqld.sock'
-default['travis_build_environment']['packer_url'] = \
-  'https://releases.hashicorp.com/packer/1.3.2/packer_1.3.2_linux_amd64.zip'
+default['travis_build_environment']['packer_version'] = '1.3.3'
 default['travis_build_environment']['packer_checksum'] = \
-  '5e51808299135fee7a2e664b09f401b5712b5ef18bd4bad5bc50f4dcd8b149a1'
-default['travis_build_environment']['packer_version'] = '1.3.2'
+  'efa311336db17c0709d5069509c34c35f0d59c63dfb05f61d4572c5a26b563ea'
 if node['kernel']['machine'] == 'ppc64le'
```

Comment:
I'm realizing now that this condition check is also not great :neutral_face: These attributes should be decided at recipe run time, not at resource load time.

Reply:
Good point! Addressed.